modules/services/arrpc.nix

```diff
@@ -0,0 +1,31 @@
+{ config, pkgs, lib, ... }:
+let
+  inherit (lib) mkIf mkPackageOption mkEnableOption;
```

Comment:
I'd stick to "with lib;" many HM modules use this

Reply:
I believe `with lib; is discouraged practice - and there's no downside to inheriting functions from the lib.

Reply:
All good, just comparing with other modules I've seen :)